device/cuda/src/seeding/spacepoint_binning.cu

```diff
@@ -56,7 +56,6 @@ sp_grid_buffer spacepoint_binning::operator()(
 
     // Helper object for the data management.
     vecmem::copy copy;
-
```

Comment:
Is there a reason for these whitespace changes?

Reply:
Result of debugging.